sdk/identity-v2/azure-identity/src/main/java/com/azure/v2/identity/implementation/client/PublicClient.java

```diff
@@ -190,9 +189,10 @@ PublicClientApplication getClient(boolean enableCae) {
                     .setAllowUnencryptedStorage(tokenCachePersistenceOptions.isUnencryptedStorageAllowed())
                     .setName(tokenCachePersistenceOptions.getName());
                 builder.setTokenCacheAccessAspect(tokenCache);
-            } catch (Throwable t) {
```

Comment:
This layer again, can see Thorwables coming from Native API access issues.

Reply:
Yeah, I will run some tests, and add the required catch for the specific Error, that may pass through. 
We will need to catch it in this case, as it will prevent DAC from executing its chain.